app/Providers/AppServiceProvider.php

```diff
@@ -23,6 +23,9 @@ public function register()
      */
     public function boot()
     {
-        //
+        if (config('app.env') === 'production') {
+            \URL::forceScheme('https');
```

Comment:
@johannasjo did we need this? Wasn't the secure_assets function enough? 🤔

Reply:
I believe the secure_assets was enough :)